torch/lib/THPP/tensors/THCTensor.hpp

```diff
@@ -149,7 +150,7 @@ struct THCTensor : public interface_traits<real>::tensor_interface_type {
   THCTensor& resize(const iterator& size_begin, const iterator& size_end,
                    const iterator& stride_begin, const iterator& stride_end);
 
-protected:
+public:
```

Comment:
There's a `private:` above for resize and it seems nice to group functions and variables in the struct definition.